codebuild/bin/grep_simple_mistakes.sh

```diff
@@ -224,6 +224,65 @@ if [[ -n $S2N_ENSURE_WITH_INVALID_ERROR_CODE ]]; then
   printf "$S2N_ENSURE_WITH_INVALID_ERROR_CODE\n\n"
 fi
 
+#############################################
+# Assert tests don't specify the "default" security policy.
+#
+# Since the "default" policies are subject to change, tests should instead specify
+# an immutable numbered policy to avoid unwanted testing behavior.
+#############################################
+S2N_DEFAULT_SECURITY_POLICY_USAGE=$(find "$PWD" -type f -name "s2n*.c" -not -path "*/tls/*" \
+    -not -path "*/bindings/*" -not -path "*/bin/*")
```

Comment:
Would this work?
```suggestion
S2N_DEFAULT_SECURITY_POLICY_USAGE=$(find "$PWD" -type f -name "s2n*.c" -path "*/tests/*")
```

Reply:
If I have built rust projects locally I get errors [1]. It might not be an issue in CI but its nice to have reproducible scripts.

I can simplify it a little to just ignore bindings:
`S2N_DEFAULT_SECURITY_POLICY_USAGE=$(find "$PWD" -type f -name "s2n*.c" -path "*/tests/*" -not -path "*/bindings/*")`


[1] `/home/toidiu/projects/s2n-tls/bindings/rust/bench/target/s2n-tls/tests/unit/s2n_self_talk_key_log_test.c`